configuration/display_rotation.md

```diff
@@ -0,0 +1,35 @@
+## How to rotate your display
+
+The options to rotate the display of your Raspberry Pi depend on which display driver software it is running, which may also depend on which Raspberry Pi you are using. 
+
+### Fake or Full KMS graphics driver (Default on Pi4)
+
+If you are running the Raspberry Pi desktop then rotation is achieved by using the `Screen Configuration Utility` from the desktop `Preferences` menu. This will bring up a graphical display of the display or displays attached. Right click on the display you wish to rotate and select the required option.
+
+It is also possible to change these settings using the command line `xrandr` option.
+
+```bash
+DISPLAY=:0 xrandr --output HDMI1 --rotate left 
+DISPLAY=:0 xrandr --output HDMI1 --rotate inverted
+DISPLAY=:0 xrandr --output HDMI1 --rotate normal
+```
+
+If you are using the console only (no graphical desktop) then you will need to set the appropriate kernel command line flags. Change the console settings as described on the [this page](./cmdline-txt.md).
+
+### Legacy graphics driver (default on models prior to the Pi4)
```

Comment:
For people who aren't sure which display driver they're using, perhaps it's worth documenting how they can check?

Reply:
To check this is a horrible three line grep command, not sure I want to submit people to it! Apparently...

grep -s -q okay /proc/device-tree/soc/v3d@7ec00000/status || grep
-s -q okay /proc/device-tree/soc/firmwarekms@7e600000/status || grep
-s -q okay /proc/device-tree/v3dbus/v3d@7ec04000/status

Reply:
Urk! :vomiting_face: 
@XECDesign Maybe a little script to do that (i.e. detect which display driver is being used) could be added to Raspbian?